packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/page.tsx

```diff
@@ -1,5 +1,10 @@
+import { Metadata } from "next";
 import { Explorer } from "./Explorer";
 
+export const metadata: Metadata = {
+  title: "Explore",
```

Comment:
this is fine for now but would be nice to have this be something like 

> Explore - 0x1234...5678 on Redstone

(I don't think we need "World Explorer" in every title)

Reply:
Yeah agreed, would be nice once we have world names available. And removed "World Explorer" from every title.